examples/13_regions.cpp

```diff
@@ -0,0 +1,261 @@
+#include <openPMD/regions/Regions.hpp>
+
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <random>
+#include <vector>
+
+////////////////////////////////////////////////////////////////////////////////
+
+void point_example() {
+  using namespace openPMD::Regions;
+  using namespace std;
+
+  cout << "\n"
+       << "Points (with dimension known at compile time):\n";
+
+  cout << "  Define two points:\n";
+  Point<int, 2> x{1, 2}, y{4, 5};
```

Comment:
Just curious: For fully known values, do they support `constexpr` construction, too?

Reply:
Yes, the API where the dimensionality is known at compile time (e.g. this example here) has constexpr constructors and operators.